Changes to be committed:

marqueswill opened the pull request:
new file:   retype.yml
	new file:   stack/next-auth/callbacks.md
	renamed:    stack/next-auth/explicacao.md -> stack/next-auth/first-steps.md
	deleted:    stack/next-auth/implementacao.md
	new file:   stack/next-auth/providers.md
	new file:   stack/next-auth/routes.md
	new file:   stack/next-auth/session.md

artistrea reviewed (changes requested):
A ideia dessa documentação é dar um "Quick Start" pra quem quiser só pegar o componente e usar logo ele. Por isso, vale a pena alterar, nesta documentação, os exemplos que ficaram mais complexos.

Recomendo, além de considerar as alterações que sugeri ao longo do código:
- [ ] colocar uma seção de como adicionar cada componente a um projeto novo.
Exemplo com o Modal:
```md
Para adicionar ao projeto esse componente, copie os seguintes arquivos para o seu projeto:
[Modal/index.tsx](https://github.com/StructCE/our-react-components/blob/main/src/components/Modal/index.tsx)
[Modal/styles.module.css](https://github.com/StructCE/our-react-components/blob/main/src/components/Modal/styles.module.css)
```

- [ ] Adicionar um arquivo explicando o que é essa seção de componentes da documentação,
    - [ ] Linkando o projeto no github;
    - [ ] E linkando o projeto no stackblitz: https://stackblitz.com/github/StructCE/our-react-components/tree/main?file=README.md,src%2FApp.tsx

JoaoS4ntos commented on Diretoria Projetos/components/RadioInput.md:
Mudei o exemplo, agr está mais prático e reduzido

JoaoS4ntos commented on Diretoria Projetos/components/Table.md:
mudei esse exemplo tambem, acho q ficou melhor

marqueswill committed:
Changes to be committed:
	new file:   retype.yml
	new file:   stack/next-auth/callbacks.md
	renamed:    stack/next-auth/explicacao.md -> stack/next-auth/first-steps.md
	deleted:    stack/next-auth/implementacao.md
	new file:   stack/next-auth/providers.md
	new file:   stack/next-auth/routes.md
	new file:   stack/next-auth/session.md

pafev commented on retype.yml:
provavelmente vc tá rodando retype start na raiz do repositório. Recomendo revisitar o readme

pafev replied:
aí apagar esse arquivo retype.yml

marqueswill committed:
Changes to be committed:
	deleted:    stack/next-auth/index.yml

marqueswill committed:
Changes to be committed:
	new file:   stack/next-auth/index.yml
	deleted:    stack/retype.yml

marqueswill committed:
Changes to be committed:
	deleted:    retype.yml
	new file:   stack/retype.yml
